content/docs/03-premium.md

```diff
@@ -1,49 +1,39 @@
 
-:::note
-If you're a Patron then you need to have your Discord account connected to your Patreon to receive your rewards. Read more on [this guide](https://support.patreon.com/hc/en-us/articles/212052266-Get-my-Discord-role#h_21f22930-84c5-4950-b6b1-3e83312f66dc). It might take some time for Patreon to update your roles, so just be patient. If you're a Patron and you're not receiving your rewards after some time, please let us know and we'll help you out!
-:::
+Please note that you must be in the support server to get your rewards, as our automated credits system requires it. If you're a Patron, you need to have your Discord account connected to your Patreon to receive your rewards. Read more about that on [this guide](https://support.patreon.com/hc/en-us/articles/212052266-Get-my-Discord-role#h_21f22930-84c5-4950-b6b1-3e83312f66dc). It might take some time for Patreon to update your roles, so please be patient. If you're a Patron and you're not receiving your rewards after some time, please let us know and we'll help you out.
 
 
-### Add Premium to your server {#how-to-add}
+### Add Premium to Your Server {#how-to-add}
 
-In the support server, you should now have access to the commands `/server add` and `/server remove`. Using these commands, you can add the server(s) you want to be granted Premium access. You don't need to be the server owner or anything, you don't even need to be in the server you want to give Premium to. Just know that if you add someone else's server, they will get access to Premium and use up one of your server slots.
+In the support server, you'll have access to the `/server add` and `/server remove` commands. Using these commands, you can add the server(s) you want to be granted Premium access. You don't need to be the server owner or anything, and you don't even need to be in the server you want to give Premium to. Just know that if you add someone else's server, they will get access to Premium and use up one of your server slots.
 
-After you've done that, you can ask one of the support members for an invite link.
+After you've done that, you can ask one of the support members for an invite link. You can list all your servers by right-clicking your profile, going to Apps in the dropdown, and then clicking "See premium information". If you're unsure how to do this, please let us know.
 
-You can list all your servers by right-clicking your profile, going to Apps in the dropdown and then click "See premium information". If you're unsure on how to do this then let us know.
 
+## Refunds Policy {#refunds}
 
-## Refunds policy {#refunds}
+Please note that there are no refunds for Premium. Premium is only $1/month, so refunding you will be a waste of time for both of us. If you're unsure about sacrificing a dollar to at least test out Premium's features, then you should probably just keep your money instead.
```

Comment:
this is huge snarky for my taste but also girlbossing

Reply:
image.png